src/messages/generic_api.py

```diff
@@ -32,14 +32,10 @@ def send_message(self, msg: str):
     def _send(self, msg: str, loot = False):
         msg = f"{Config().general['name']}: {msg}"
 
-        if loot:
-            url = Config().general['custom_loot_message_hook']
-            if not url:
-                url = Config().general['custom_message_hook']
-        else:
-            url = Config().general['custom_message_hook']
-        if not url:
+        if not (url := Config().general['custom_message_hook']):
             return
+        if loot and (loot_url := Config().general['custom_loot_message_hook']):
+            url = loot_url
```

Comment:
I simplified the logic here. Does the difference make sense to you? The walrus operator := is great for this sort of scenario--if you haven't learned about it yet check it out.

Reply:
Much better than mine. :)